### DISCUSSION TITLE
malwarebyte flagged this file (CLANGARM.exe) as malware on my windows 10 laptop. Is this a false positive?

### DISCUSSION THREAD

**Discussion by sandeep026** [msys2/MSYS2-packages#4616] [Q&A]
![image](https://github.com/msys2/MSYS2-packages/assets/16457676/61fc26df-ac96-43e8-9ef0-e655294a4c92)

**Comment by Biswa96** [comment]
Yes, that is false positive. That executable is compiled from https://github.com/msys2/msys2-launcher repository using https://github.com/msys2/MSYS2-packages/blob/master/msys2-launcher/PKGBUILD build script.
